bin/reset.digital-loggers

```diff
@@ -20,3 +20,5 @@
 
 curl --data ${power_port}=CCL -o /dev/null --silent \
 	http://${power_user:-admin}:${power_pass:-1234}@${power_ip}/outlet
+
+[ ! -z "${recovery_impl}" ] && . "${bin_dir}/recovery.${recovery_impl}" || true
```

Comment:
While I'm happy to see the ?: operator in expressions, I'd far rather see a full if []; then ...; fi for code-flow.